qiita_pet/handlers/admin_processing_job.py

```diff
@@ -105,28 +105,70 @@ def get(self):
 
     @execute_as_transaction
     def post(self):
-
-        # Get user-inputted qiita id and sample names
+        # Get user-inputted qiita id and sample names / tube_ids
         qid = self.get_argument("qid")
         snames = self.get_argument("snames").split()
 
+        # Get study give qiita id
+        st = Study(qid).sample_template
```

Comment:
st is a not a properly descriptive name for a variable, especially one that plays an important role some ten lines later in the code.

Reply:
Would changing the variable name to `study` work better, or would a variable name like `qt_study` be more appropriate?

Reply:
study is fine.